front/src/components/moodboard/moodItemsForAccessibility.component.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from "react";
+import { useCallback, useState } from "react";
+import { StyleSheet, View } from "react-native";
+import { Text } from "react-native-elements";
+import { TouchableOpacity } from "react-native-gesture-handler";
+
+import { Colors, Margins, Paddings } from "../../styles";
+import { MOODBOARD_ITEMS } from "../../utils/moodboard.util";
+
+interface Props {
+  setActiveIndex: any;
```

Comment:
```suggestion
  setActiveIndex: Dispatch<SetStateAction<number>>;
```

En ajoutant l'import `import type { Dispatch, SetStateAction } from "react";`

Dans la mesure du possible, il faut éviter les `any` car c'est trop permissif